app/views/markets/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1><%= "#{@market.name.gsub(/[A-Za-z']+/,&:capitalize)}" %></h1>
```

Comment:
This would be good to move into the controller.  It's good practice to put any logic should go into the controller.